demisto_sdk/commands/format/update_generic_yml.py

```diff
@@ -5,13 +5,14 @@
 import click
 from ruamel.yaml import YAML
 
-from demisto_sdk.commands.common.constants import (INTEGRATION, PLAYBOOK,
+from demisto_sdk.commands.common.constants import (ENTITY_TYPE_TO_DIR,
+                                                   INTEGRATION, PLAYBOOK,
                                                    TEST_PLAYBOOKS_DIR,
                                                    FileType)
-from demisto_sdk.commands.common.tools import (_get_file_id, find_type,
-                                               get_entity_id_by_entity_type,
-                                               get_not_registered_tests,
-                                               get_yaml, is_uuid)
+from demisto_sdk.commands.common.tools import (
+    _get_file_id, find_type, get_entity_id_by_entity_type,
+    get_not_registered_tests, get_scripts_and_commands_from_yml_data, get_yaml,
+    is_uuid)
```

Comment:
probably need to run precommit after (sorry :( )
```suggestion
    is_uuid, listdir_fullpath)
```